File: bhaskara.c
#include <stdio.h>
#include <string.h>
#include <math.h>

int main(){

    int n;

    scanf ("%d", &n);

    for (int i=0;i<n;i++){

        int a,b,c;

        scanf ("%d", &a);
        scanf ("%d", &b);
        scanf ("%d", &c);

        float bhaskara= b*b -4*a*c;

        int podeCalcular=1;
        if (bhaskara <0){

            printf ("Impossivel Calcular\n");
            podeCalcular=0;
        }

        if (podeCalcular==1){
        float raiz1= -b + sqrt(bhaskara) / 2*a;
        float raiz2= b - sqrt(bhaskara) / 2*a;

        printf ("%f\n", raiz1);
        printf ("%f\n", raiz2);
    }
    }

    return 0;
}

File: bhaskara.java
import java.util.Scanner;

public class bhaskara {

    public static void main(String[] args) {
        
        Scanner scanner = new Scanner(System.in);

        int n = scanner.nextInt();
       

        for (int i = 0; i < n; i++) {

            double a = scanner.nextDouble();
            double b = scanner.nextDouble();
            double c = scanner.nextDouble();

            double bhaskara = b * b - 4 * a * c;

            if (bhaskara < 0) {
                System.out.println("Impossivel Calcular");
            } else {
                double raiz1 = (-b + Math.sqrt(bhaskara)) / (2 * a);
                double raiz2 = (-b - Math.sqrt(bhaskara)) / (2 * a);

                System.out.printf("%.5f", raiz1);
                System.out.printf("%.5f", raiz2);
            }
        }
    }
}
